client/src/components/BoardObject.tsx

```diff
@@ -127,6 +131,38 @@ export const BoardObject = (props: BoardObjectProps) => {
 					/>
 				)}
 				{type === 'map' && data && <Map />}
+				{type === 'chat' && chat && (
+					<div> <Button style={{ maxWidth: 300, maxHeight: 40 , minWidth: 300, minHeight: 40, fontSize: 20 }} size="medium" color="primary"> Chat</Button>
```

Comment:
Not sure why this is a button but it's misleading since clicking it does nothing